public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -174,7 +174,10 @@ function Main (params) {
         );
 
         // Speed limit
-        svl.speedLimit = new SpeedLimit(svl.panorama, svl.map.getPosition, svl.isOnboarding, null);
+        // Checking to see if the labeltype is NoCurbRamp before adding speedlimit.
+        if (currLabel.labelType == 'NoCurbRamp') {
```

Comment:
This is code for the Explore page. Please make sure any time you're adding code, that the code you've added actually does something! Since this doesn't make changes to the page you thought it did, you would have thought that it did nothing and investigated further. What it actually does it break the Explore page so that it doesn't load at all :)